src/components/BuildConfigRevisionsList/BuildConfigRevisionsList.tsx

```diff
@@ -32,12 +37,15 @@ export const BuildConfigRevisionsList = ({
 }: IBuildConfigRevisionsListProps) => {
   const navigate = useNavigate();
   const { search } = useLocation();
+  const responsiveStyle: CustomCSSProperties = {
+    '--pf-c-tabs--m-vertical--MaxWidth': 'none',
+  };
```

Comment:
- this object should not be declared here, put that into the CSS file
- change the max-width CSS property rather than PF var definition

Reply:
fixed